modules/spring-data-2.0/src/test/java/org/apache/ignite/springdata/misc/ApplicationConfiguration.java

```diff
@@ -34,6 +34,16 @@
 @EnableIgniteRepositories
 public class ApplicationConfiguration {
 
+    /**
+     * The bean with cache names
```

Comment:
Need dots in the end :)

```suggestion
     * The bean with cache names.
```